Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -0,0 +1,1764 @@
+        return response
+
+    def get_detected_exploit(self, detected_exploit_id: str) -> Dict:
+        """
+        Get a single detected exploit.
+
+        Args:
+            detected_exploit_id (str): ID of the exploit to return.
+
+        Returns:
+            response (Response): API response from Sophos.
+        """
+        url_suffix = 'endpoint/v1/settings/exploit-mitigation' \
+                     f'/detected-exploits/{detected_exploit_id}'
+        response = self._http_request(method='GET', headers=self.headers,
+                                      url_suffix=url_suffix)
+        return response
+
+
+def get_jwt_token(client_id: str, client_secret: str) -> dict:
+    """
+    Send an auth request to Sophos Central and receive an access token.
+
+    Args:
+        client_id (str): Sophos Central client id.
+        client_secret (str): Sophos Central client secret.
+
+    Returns:
+        response.json (dict): API response from Sophos.
+    """
+    headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+    body = {
+        'grant_type': 'client_credentials',
+        'client_id': client_id,
+        'client_secret': client_secret,
+        'scope': 'token'
+    }
+    response = post(headers=headers, data=body,
+                    url='https://id.sophos.com/api/v2/oauth2/token')
+    if not response.ok:
+        raise DemistoException(response.text)
+    return response.json()
+
+
+def get_client_data(bearer_token: str):
+    """
+    Set the tenant ID required for the API endpoints. Updates the existing self.headers.
+
+    Args:
+        bearer_token (str): A JWT token.
+
+    Returns:
+        headers (dict): Headers for the client.
+        base_url (str): Base URL for the client.
+    """
+    headers = {'Authorization': 'Bearer ' + bearer_token}
+    response = get(headers=headers,
+                   url='https://api.central.sophos.com/whoami/v1')
+    try:
+        response_data = response.json()
+    except (json.JSONDecodeError, DemistoException) as exception:
+        raise DemistoException('Failed getting a whoami response JSON: ', exception) from exception
+    headers.update({'X-Tenant-ID': response_data.get('id')})
+    api_hosts = response_data.get('apiHosts')
+    if not api_hosts:
+        raise DemistoException('Error finding data region. ' + str(response))
+    base_url = api_hosts.get('dataRegion') + '/'
+    return headers, base_url
+
+
+def create_alert_output(item: Dict, table_headers: List[str]) -> Dict[str, Optional[Any]]:
+    """
+    Create the complete output dictionary for an alert.
+
+    Args:
+        item (dict): A source dictionary from the API response.
+        table_headers (list(str)): The table headers to be used when creating initial data.
+
+    Returns:
+        object_data (dict(str)): The output dictionary.
+    """
+    object_data = {field: item.get(field) for field in table_headers + ['groupKey', 'product']}
```

Comment:
please rename all these vars in the func
```suggestion
    alert_data = {field: item.get(field) for field in table_headers + ['groupKey', 'product']}
```